resolwe/flow/executors/docker.py

```diff
@@ -38,6 +38,10 @@ def start(self):
             data_id = 'test_{}'.format(random.randint(1000, 9999))
         command_args['container_name'] = '--name=resolwe_{}'.format(data_id)
 
+        # configure Docker network mode for the container (if specified)
```

Comment:
Start comments with a capital letter. Also write what is the default networking mode that is used here. From PEP8:
> Comments should be complete sentences. If a comment is a phrase or sentence, its first word should be capitalized, unless it is an identifier that begins with a lower case letter (never alter the case of identifiers!).